doc/sphinx/source/input.rst

```diff
@@ -123,28 +123,51 @@ At the moment, cmorize_obs supports Python and NCL scripts.
 
 Cmorization as a fix
 --------------------
-As of early 2020, ESMValTool also provides (limited) support for data in their native format. In this case, the steps needed to reformat the data are executed as datasets fixes during the execution of an ESMValTool recipe, as one of the first preprocessor steps. Compared to the workflow described above, this has the advantage that the user does not need to store a duplicate (cmorized) copy of the data. Instead, the cmorization is performed 'on the fly' when running a recipe. ERA5 is the first dataset for which this 'cmorization on the fly' is supported.
-
-To use this functionality, users need to provide a path for the ``native6`` project data in the :ref:`user configuration file<config-user>`. Then, in the recipe, they can refer to the native6 project, like so:
+As of early 2020, ESMValTool also provides support for some data in their native
```

Comment:
```suggestion
ESMValCore also provides support for some datasets in their native
```
I think the date reference can be removed, users are mostly interested in the current features, if they want the history they can look at the changelog. It would also be good to distinguish between ESMValTool and ESMValCore. The native support is an ESMValCore feature.